src/main/java/frc/lib/team3061/vision/Vision.java

```diff
@@ -31,6 +31,7 @@ public class Vision extends SubsystemBase {
 
   private double lastTimestamp;
   private SwerveDrivePoseEstimator poseEstimator;
+  private boolean vision_subsystem_is_enabled;
```

Comment:
Since this instance variable is part of the vision subsystem, its name doesn't need to reflect that. isEnabled would suffice. It should be initialized in the constructor to true.

Also check the warnings in VSCode. Underscores are not recommended for identifiers.

Reply:
done